lte/gateway/configs/pipelined.yml

```diff
@@ -164,6 +164,12 @@ he_proxy_eth_mac: 'e6:8f:a2:80:80:80'
 # Make as a True when gnb ip will support for uplink 
 ovs_multi_tunnel: False
 
+# Internal port for processing internal conntrack
+ovs_internal_conntrack_port_number: 15579
```

Comment:
@prabinakpattnaik  Add this to pipelined yml prod

Reply:
Sure. I will add both variables into prod file.

Reply:
changes done